content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+यह पेज React के दस्तावेज और उससे सम्बंधित साधनो का ओवरव्यू है |
 
-**React** is a JavaScript library for building user interfaces. Learn what React is all about on [our homepage](/) or [in the tutorial](/tutorial/tutorial.html).
+**React** एक जावास्क्रिप्ट लाइब्रेरी है जो उपयोगकर्ता को इंटरफेसेस बनाने में काम आती है | [हमारे होमपेज](/) पर या [इस टुटोरिअल](/tutorial/tutorial.html) पर सीखें की React किस चीज़ के बारे में है |
 
----
-
-- [Try React](#try-react)
-- [Learn React](#learn-react)
-- [Staying Informed](#staying-informed)
-- [Versioned Documentation](#versioned-documentation)
-- [Something Missing?](#something-missing)
-
-## Try React {#try-react}
+ ---
+ 
+ - [React का प्रयोग करके देखें](#try-react)
+ - [React सीखे](#learn-react)
+ - [सूचित रहे](#staying-informed)
+ - [वर्ज़न दस्तावेज़](#versioned-documentation)
+ - [क्या छूटा है?](#something-missing )
+ 
+## React का प्रयोग करके देखें {#try-react}
 
-React has been designed from the start for gradual adoption, and **you can use as little or as much React as you need.** Whether you want to get a taste of React, add some interactivity to a simple HTML page, or start a complex React-powered app, the links in this section will help you get started.
+React को धीरे-धीरे ग्रहण करने के लिए बनाया गया है, और  **आप जितनी कम और ज्यादा चाहे उतनी React इस्तेमाल कर सकते है |** यदि आप React का नमूना देखना चाहें, अपने सरल HTML पेज में कुछ इंटरएक्टिविट डालना चाहें, या एक जटिल React-संचालित ऐप बनाना चाहे, तो इसके लिए इस अनुभाग में दी गई लिंक्स आपकी सहायता करेंगी | 
```

Comment:
React-संचालित ऐप बनाना चाहे,  तो इसके लिए इस अनुभाग में दी गई लिंक्स आपकी सहायता करेंगी |  -> React-संचालित ऐप बनाना चाहे, in sabhi ke liye इस अनुभाग में दी गई लिंक्स आपकी सहायता करेंगी